conda/testing/gateways/fixtures.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+
+import socket
+import tempfile
+from pathlib import Path
+
+import pytest
+
+from .helpers import have_minio
+
+
+def s3_server(xprocess):
```

Comment:
```suggestion
def minior_s3_server(xprocess, tmp_path):
```

Let's rename this fixture to `minio_s3_server`, since we know it's not really the same as AWS S3 and it's worth pointing that out. And also pass in `tmp_path` for later use.